samples/cellular/nrf_cloud_multi_service/Kconfig

```
@@ -154,19 +154,20 @@ choice
 	default LED_INDICATION_GPIO if BOARD_NRF9160DK_NRF9160_NS || BOARD_NRF9161DK_NRF9161_NS || $(dt_compat_enabled,gpio-leds)
 	default LED_INDICATION_DISABLED
 
-	config LED_INDICATION_PWM
-		select LED
-		select LED_PWM
-		select PWM
-		bool "Enable LED indication using the pwm-leds driver"
-
-	config LED_INDICATION_GPIO
-		select LED
-		select LED_GPIO
-		bool "Enable LED indication using the gpio-leds driver"
-
-	config LED_INDICATION_DISABLED
-		bool "Disable LED indication"
+config LED_INDICATION_PWM
+	select LED
+	select LED_PWM
+	select PWM
+	bool "Enable LED indication using the pwm-leds driver"
```

```suggestion
	bool "Enable LED indication using the pwm-leds driver"
	select LED
	select LED_PWM
	select PWM
```
type is always first